tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -787,11 +787,12 @@ def test_with_orphan_trace(self):
                     "trace_id": self.trace_id,
                 }
             ],
-            # Some random id so its separated from the rest of the trace
             parent_span_id=orphan_span_ids["root_span"],
             span_id=orphan_span_ids["child"],
             project_id=self.gen1_project.id,
-            duration=500,
+            # Because the snuba query orders based is_root then timestamp, this causes grandchild1-0 to be added to
+            # results first before child1-0
+            duration=2500,
```

Comment:
This is some glass-box testing, cause i know doing this will break this test without the changes from this PR in `src/sentry/api/endpoints/organization_events_trace.py`

When I refactor this endpoint later I'll make cases like this easier to test for.